code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -191,45 +191,28 @@
 		return
 
 	locked = TRUE //lock gibber
+	occupant.dir = SOUTH
 
-	var/image/gibberoverlay = new //used to simulate 3D effects
-	gibberoverlay.icon = icon
-	gibberoverlay.icon_state = "grinder_overlay"
-	gibberoverlay.overlays += image('icons/obj/kitchen.dmi', "grinder_idle")
-	icon_state = "grinder_on"
-
-	var/image/feedee = new
-	occupant.dir = 2
-	feedee.icon = getFlatIcon(occupant, 2) //makes the image a copy of the occupant
-
-	var/atom/movable/holder = new //holder for occupant image
-	holder.name = null //make unclickable
-	holder.overlays += feedee //add occupant to holder overlays
-	holder.pixel_y = 25 //above the gibber
-	holder.loc = get_turf(src)
-	holder.layer = MOB_LAYER //simulate mob-like layering
-	holder.anchored = TRUE
-
-	var/atom/movable/holder2 = new //holder for gibber overlay, used to simulate 3D effect
-	holder2.name = null
-	holder2.overlays += gibberoverlay
-	holder2.loc = get_turf(src)
-	holder2.layer = MOB_LAYER + 0.1 //3D, it's above the mob, rest of the gibber is behind
-	holder2.anchored = TRUE
-
-	animate(holder, pixel_y = 16, time = animation_delay) //animate going down
-
+	var/image/victim = image(icon = getFlatIcon(occupant, 2), loc = src, layer = MOB_LAYER, pixel_y = 25) // Copies the look of the victim and places them above the gibber, can't grab the exact icon because it needs to be considered as static to mask the feet later on
```

Comment:
```suggestion
	var/image/victim = image(icon = getFlatIcon(occupant, SOUTH), loc = src, layer = MOB_LAYER, pixel_y = 25) // Copies the look of the victim and places them above the gibber, can't grab the exact icon because it needs to be considered as static to mask the feet later on
```